content/docs/user-guide/networking/multiplayer/overview.md

```diff
@@ -45,6 +45,30 @@ Then, clients update themselves from a delta by:
 
 After, the AzNetworking layer becomes aware that the world was updated. This occurs automatically through ACK vector replication, and does not require any further action from the client.
 
+### Types of network properties
+
+Network properties have two important fields: `ReplicateFrom` and `ReplicateTo`. 
+Together, these define which role can replicate to which role. You can only replicate property values *from* Authority and Autonomous roles. Under the server-authority model of O3DE, this ensures that you never accidentally replicate from an unprivileged client. 
```

Comment:
Actually this is about only replicating from roles that have write access to component data. If you only have read access (no controller), then you can't change any values, therefore there could never actually be anything for that role to replicate.

Reply:
```suggestion
Together, these define which role can replicate to which role. You can only replicate property values *from* Authority and Autonomous roles. A `ReplicateFrom` Authority relationship creates a server-authority model, which ensures that you never accidentally replicate from an unprivileged client. 
```

Reply:
That suggestion looks good. It may be good to note that ReplicateFrom Autonomous can be useful for sending non-authoritative data like telemetry.